src/scripttabs.cpp

```diff
@@ -108,7 +108,7 @@ TYPE_SYMBOL asTypeTable[] =
 	{ "",	(INTERP_TYPE)ST_POINTER_STRUCTSTAT, AT_SIMPLE, NULL, NULL },	//for NULLSTRUCTURESTAT
 
 	/* This final entry marks the end of the type list */
-	{ "END OF TYPE LIST", (INTERP_TYPE)NULL,		AT_SIMPLE,	NULL,	NULL },
+	{ "END OF TYPE LIST", (INTERP_TYPE)0,		AT_SIMPLE,	NULL,	NULL },
```

Comment:
Have you tried using NULL directly? I.e. removing the cast instead of this? Because your change seems to be the opposite of the intent for introducing `nullptr` in the language...